### ISSUE TITLE
An error with Xcode9/CP

### ISSUE THREAD

**Issue by icanzilb** [realm/realm-cocoa#5011]
RealmSwift (via CocoaPods) is ready to use on Xcode9 b1 when its target is set to Swift 3.2 except for one issue:

![image](https://user-images.githubusercontent.com/633862/26967832-03d7a166-4d00-11e7-8535-cc04d7feb107.png)

Once the "Fix" button is clicked it's possible to create a Swift 4 app with the modified current version of RealmSwift. Maybe it's worth making the change to the code right now so people can start testing their apps in Xcode9?

**Comment by tgoyne** [comment]
This was fixed by https://github.com/realm/realm-cocoa/pull/5006.

**Comment by hishnash** [comment]
@tgoyne is it possible to install this through coco-pods
```
pod 'RealmSwift', :git => 'https://github.com/realm/realm-cocoa.git', :branch => 'master'
```
this does not work cocopods fails to download.

**Comment by feighter09** [comment]
@tgoyne bump, this was *not* fixed by #5006 for Cocoapods users

**Comment by austinzheng** [comment]
What sort of errors are you getting? If Xcode is complaining about missing files, it's because you need to add `:submodules => true` to the pod definition.
